### PR TITLE
Add support for Default Parameter Value

### PR DESCRIPTION
**Pull request by BenjaminVanRyseghem**
When adding a default value to the `initialize` method, the
`require-class-comment` rule failed.

---
This change is [Reviewable](https://reviewable.io/reviews/foretagsplatsen/ftgp-eslint/21)

### CODE REVIEW

File: tests/fixtures/rules/require-class-comment/fixtures.js [hunk 1]
```diff
@@ -1,59 +1,68 @@
...
-		' * @param {string} spec.bar - identifier\n with long description\n' +
-		' * @param {number} spec.baz - number\n' +
-		' */\n' +
-		'model.subclass(function(that, my){\n' +
-		'	my.initialize = function(spec) {\n' +
-		'		my.bar = spec.bar\n' +
-		'		my.baz = spec.baz\n' +
-		'	};\n' +
-		'})',
+			' * Foo\n' +
```
Comment (DamienCassou) [inline tests/fixtures/rules/require-class-comment/fixtures.js]: Can you please change formatting in a separate commit so we better see what has changed?

File: tests/fixtures/rules/require-class-comment/fixtures.js [hunk 2]
```diff
@@ -1,59 +1,68 @@
...
+			' * @param {number} [spec.baz] - number\n' +
+			' */\n' +
+			'model.subclass(function(that, my){\n' +
+			'	my.initialize = function(spec) {\n' +
+			'		my.baz = spec.baz\n' +
+			'	};\n' +
+			'})',
+		paramsWithDefaultValue: '/**\n' +
+			' * Foo\n' +
+			' * @param {string} spec.bar - identifier\n with long description\n' +
```
Comment (DamienCassou) [inline tests/fixtures/rules/require-class-comment/fixtures.js]: shouldn't `spec.bar` be marked optional (`[spec.bar]`) in the comment?

Comment (NicolasPetton) [inline tests/fixtures/rules/require-class-comment/fixtures.js]: no, as there's no default value for it here.

File: tests/fixtures/rules/require-class-comment/fixtures.js (hunk 1 above)
Comment (NicolasPetton) [inline tests/fixtures/rules/require-class-comment/fixtures.js]: @DamienCassou done. Can you have another look?

File: tests/fixtures/rules/require-class-comment/fixtures.js [hunk 3]
```diff
@@ -45,15 +45,24 @@ module.exports = {
...
 		'})'
 	},
 	invalid: {
 		missingDash: {
 			code: '/**\n' +
-			' * Awesome description\n' +
-			' * @param {boolean} foo Foo\n' +
-			' */\n' +
-			'model.subclass(function(){});',
+				' * Awesome description\n' +
```
Comment (BenjaminVanRyseghem) [inline tests/fixtures/rules/require-class-comment/fixtures.js]: this commit is still re-indenting shitloads of unrelated code

Comment (NicolasPetton) [inline tests/fixtures/rules/require-class-comment/fixtures.js]: Fixed, thanks!

**Comment by NicolasPetton** [comment]
@BenjaminVanRyseghem @DamienCassou can you have another look?

**Comment by BenjaminVanRyseghem** [comment]
good for me
